networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/forks/GossipForkManager.java

```diff
@@ -246,6 +249,23 @@ private <T> void publishMessage(
                     slot));
   }
 
+  private <T> SafeFuture<Void> publishMessageWithFeedback(
+      final UInt64 slot,
+      final T message,
+      final String type,
+      final BiFunction<GossipForkSubscriptions, T, SafeFuture<Void>> publisher) {
+    final Optional<GossipForkSubscriptions> gossipForkSubscriptions =
+        getSubscriptionActiveAtSlot(slot).filter(this::isActive);
+
+    if (gossipForkSubscriptions.isEmpty()) {
+      LOG.warn(
+          "Not publishing {} because no gossip subscriptions are active for slot {}", type, slot);
+      return SafeFuture.COMPLETE;
```

Comment:
hrm this is ok as long as they're all valid i guess...

Reply:
the idea is that I don't want to expose errors up in the stack:
https://github.com/Consensys/teku/pull/8735/files#diff-95223a152d06b6a8980fc01c5acf96e98971433ecb3390c8c90a994ea5a07c48R85-R91

We could argue if this is a good approach or not...